csfieldguide/static/interactives/matrix-simplifier/js/matrix-simplifier.js

```diff
@@ -72,7 +92,8 @@ $.getScript(mjaxURL, function() {
 $(document).ready(function() {
   $('#add-matrix-from-input').click(addMatrix);
   $('#add-vector-from-input').click(addVector);
```

Comment:
Not related to the changes I think but imo the default new vector should be 0, 0, 0 so (like the new matrix) by default it doesn't change anything
![image](https://user-images.githubusercontent.com/25916475/66618044-1a95f300-ec34-11e9-9372-d544a1e3412a.png)